### PR TITLE
Adding keepalive documentation.

### PR DESCRIPTION
**Pull request by nstielau**
Hey,

I haven't checked to see how this looks (OS X yakshaving), but thought I'd save someone else some code diving for this stuff.

### CODE REVIEW

File: 0.11/keepalives.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# Keepalive Checks
+
+## What are keepalive checks?
+
+
+Keepalive checks are integrated checks that monitor the communication
+between Sensu clients and the Sensu server.
+
+Clients publish keepalive status updates periodically, which are
+processed and recorded by the server.  The server then periodically
```
Comment (grepory) [inline 0.11/keepalives.md]: processed and recorded by the server.  The server ~~then periodically~~

File: 0.11/keepalives.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+verifies that all clients have published a keepalive within the
+given thresholds, and escalates if there has been no client-server
```
Comment (grepory) [inline 0.11/keepalives.md]: within a configurable threshold, and _it_ escalates

File: 0.11/keepalives.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+communication.
+
+Keepalive checks handle basic monitoring scenarios, such as server
+failure, excessive load, client misconfiguration, and network issues.
+
+## Keepalive default configuration
+
+Keepalive checks escalate via the 'default' handle set, and issue a
```
Comment (grepory) [inline 0.11/keepalives.md]: handler set, ~~and~~ issue a warning...

File: 0.11/keepalives.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+warning after 120 seconds without communication, and issue a critical
+event after 180 seconds. Due to the periodic nature of both the
+publishing and verification, there may be a few seconds of addition
```
Comment (grepory) [inline 0.11/keepalives.md]: additional

File: 0.11/keepalives.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+time before the events are created and handled.
+
+## Overriding Configuration
+
+The out-of-the-box keepalive configuration is meant to be a workable
```
Comment (grepory) [inline 0.11/keepalives.md]: The default keepalive configuration is meant to be sufficient, however these settings can be overridden...

**Comment by grepory** [comment]
Various style and grammar edits. Also, thanks!

**Comment by nstielau** [comment]
Fixed, thanks!

On Mon, Oct 14, 2013 at 1:24 PM, Greg Poirier notifications@github.comwrote:

> Various style and grammar edits. Also, thanks!
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/sensu/sensu-docs/pull/44#issuecomment-26285766
> .

## 

☮
Nick Stielau
206-245-7991

**Comment by jamtur01** [comment]
Awesome stuff! :+1:

**Pull request merged**